torch_frame/__init__.py

```diff
@@ -16,10 +16,11 @@
     'numerical',
     'categorical',
     'text_embedded',
-    'multi_categorical',
+    'multicategorical',
     'TaskType',
     'NAStrategy',
     'TensorFrame',
+    'TensorData',
```

Comment:
Is there a reason to expose it? I would prefer to keep it internal only.

Reply:
Yeah, makes sense.